netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
 	publish = "dist"
-	command = "npm run build"
+	command = "node ./scripts/build.js && npm run build"
```

Comment:
Could consider renaming `build.js` to something like `createProdToken.js` to give more context